content/_data/changelogs/weekly.yml

```diff
@@ -1,7 +1,36 @@
 - version: "2.47"
-  date: 2017-02-17
+  date: 2017-02-19
   changes:
     - type: bug
       message: >
         Snapshot builds of plugins that had dependencies on other snapshot builds were not having their version numbers compared correctly
       issue: 41899
+    - type: bug
+      message: Update Groovy to 2.4.8 to address memory leak issue
+      issue: 33358
+    - type: bug
+      message: >
+        Internal: Add support for dated Maven snapshot builds in version number comparisons
+      issue: 41899
+    - type: bug
+      message: Windows service restart did not retain build queue
+      issue: 32820
+    - type: bug
+      message: Exceptions during Jenkins cleanup step should not block restart
+      issue: 42164
+    - type: bug
+      message: >
+        Remoting 3.5: Remoting clients now accept lowercase (HTTP 2) headers sent by reverse proxies
+      issue: 40710
+    - type: rfe
+      message: >
+        Remoting 3.5: Add option to specify the remoting protocol to use on the client
+      issue: 41730
+    - type: bug
+      message: >
+        Remoting 3.5: Stability improvements
```

Comment:
Maybe it's more preferable than the previous approach.
I would also add a hyperlink to the full changelog in the message then